### PR TITLE
fix: Distinguishes between actual bad-requests and "conference ID not found" errors from the bridge.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java [hunk 1]
```diff
@@ -2358,8 +2358,20 @@ private void onBridgeUp(Jid bridgeJid)
...
     /**
-     * Handles on bridge down event by shutting down the conference if it's the
-     * one we're using here.
+     * Notifies this conference that one of its channel allocators failed to
+     * allocate channels, and that the participants on a specific bridge need
+     * to be re-invited.
+     * @param bridgeJid the JID of the bridge on which participants need to be
+     * re-invited.
+     */
+    void channelAllocationFailed(Jid bridgeJid)
```
Comment (bbaldino) [inline src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: is this called anywhere?

Comment (bgrozev) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: It is now (I just fixed it)

Comment (bbaldino) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: i still don't see it?

Comment (bgrozev) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: It's here: https://github.com/jitsi/jicofo/pull/411/commits/a57d14ed382fb902ea94d0b8acdecd89579304fc#diff-653d615bc7b6b3b1f8196be7f8491c8eR313

**Pull request merged**